packages/ava/src/advisor/ruler/rules/aggregation-single-row.ts

```diff
@@ -1,8 +1,8 @@
-import { CKBJson } from '@antv/ckb';
+import { ckb } from '../../../ckb';
 
 import type { RuleModule } from '../interface';
 
-const Wiki = CKBJson('en-US', true);
+const Wiki = ckb();
```

Comment:
CKB should be param of the rule function, o/w. custom CKB could not work

Reply:
same for all rule files

Reply:
done